lexicon/providers/nfsn.py

```diff
@@ -0,0 +1,164 @@
+from __future__ import absolute_import
+from __future__ import print_function
+
+import hashlib
+import json
+import logging
+import random
+import string
+import time
+
+import requests
+
+from lexicon.providers.base import Provider as BaseProvider
+
+from urllib.parse import urlencode
+
+logger = logging.getLogger(__name__)
+
+
```

Comment:
Module needs to declare a list named `NAMESERVER_DOMAINS`. It should contain any domain used by the nfsn nameservers.

I suggest the following, based on what the documentation says about the nfsn nameservers:
`NAMESERVER_DOMAINS = ['nearlyfreespeech.net']`

See `lexicon/providers/ovh.py` for implementation example.